ext/cgi/escape/escape.c

```diff
@@ -1,5 +1,6 @@
 #include "ruby.h"
 #include "ruby/encoding.h"
+#include <stdbool.h>
```

Comment:
The cgi gem still supports ruby 2.5, which does not assume C99.
We should use `int` instead without `stdbool.h`.